landing/tests/tests.py

```diff
@@ -0,0 +1,29 @@
+import pytest
```

Comment:
Maybe make the file a bit more descriptive than `tests.py`?

Reply:
> Maybe make the file a bit more descriptive than `tests.py`?

Resolved